config/ceph.yml

```diff
@@ -7,11 +7,16 @@ products:
 - ceph.owl: https://raw.githubusercontent.com/obophenotype/cephalopod-ontology/master/ceph.owl
 - ceph.obo: https://raw.githubusercontent.com/obophenotype/cephalopod-ontology/master/ceph.obo
 
-term_browser: ontobee
+term_browser: custom
 example_terms:
-- CEPH_0000000
+- CEPH_0000256
 
 entries:
+- prefix: /about/
+  replacement: https://www.ebi.ac.uk/ols/ontologies/ceph/terms?iri=http://purl.obolibrary.org/obo/CEPH_
```

Comment:
Now that the `tools/test.py` is fixed, this fails because there ends up being an extra `CEPH_`.